packages/widgetbook/lib/src/knobs/bool_knob.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:widgetbook/src/knobs/knobs.dart';
+import 'package:widgetbook/src/knobs/nullable_checkbox.dart';
+
+class BoolKnob extends Knob<bool> {
+  BoolKnob({
+    required String label,
+    String? description,
+    required bool value,
+  }) : super(
+          label: label,
+          description: description,
+          value: value,
+        );
+
+  @override
+  Widget build() => BooleanKnobWidget(
+        label: label,
+        description: description,
+        value: value,
+      );
+}
+
+class NullableBoolKnob extends Knob<bool?> {
+  NullableBoolKnob({
+    required String label,
+    String? description,
+    required bool? value,
+  }) : super(
+          label: label,
+          description: description,
+          value: value,
+        );
+
+  @override
+  Widget build() => BooleanKnobWidget(
+        label: label,
+        description: description,
+        value: value,
+        nullable: true,
+      );
+}
+
+class BooleanKnobWidget extends StatefulWidget {
+  const BooleanKnobWidget({
+    Key? key,
+    required this.label,
+    required this.description,
+    required this.value,
+    this.nullable = false,
+  }) : super(key: key);
+
+  final String label;
+  final String? description;
+  final bool? value;
+  final bool nullable;
+
+  @override
+  State<BooleanKnobWidget> createState() => _BooleanKnobWidgetState();
+}
+
+class _BooleanKnobWidgetState extends State<BooleanKnobWidget> {
+  bool value = false;
+
+  @override
+  void initState() {
+    super.initState();
+    if (widget.value != null) {
+      value = widget.value!;
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final disabled = widget.value == null;
+    return KnobWrapper(
+      description: widget.description,
+      child: KnobWrapper(
+        description: widget.description,
+        child: ListTile(
+          title: Text(widget.label, overflow: TextOverflow.ellipsis),
+          dense: true,
+          subtitle:
+              widget.description == null ? null : Text(widget.description!),
+          trailing: Wrap(
```

Comment:
This knob looks very different to all the other knobs. In addition, `KnobWrapper` is used multiple times leading to the description being displayed three times: 

<img width="378" alt="image" src="https://user-images.githubusercontent.com/16862002/163110772-149e2e3d-132d-49ef-9c67-b998499eda69.png">

I'll propose a design suggestion to make the Panel more accessible.